sdk/python/feast/feature_store.py

```diff
@@ -1078,15 +1079,15 @@ def get_online_features(
 
         return self._get_online_features(
             features=features,
-            entity_values=columnar,
+            entity_values=dict(columnar),
```

Comment:
is this intentional?

Reply:
Yeah - two reasons:

1. I figured that passing it along as a `defaultdict` was just asking for an unhelpful error message
2. I think that `defaultdict` might not technically be a `Dict`? But might be wrong on that.

Reply:
any reason why we wouldnt just use a normal dict from the getgo then?

Reply:
Laziness 😛